### ISSUE TITLE
Headers are missing on "properties" and "contact" pages

### ISSUE THREAD

**Issue by tremby** [tremby/markwiens.ca-ubertor#56]
This might be on purpose, but it stands out negatively to me.

Properties:

![2023-08-23_21-42](https://github.com/tremby/markwiens.ca-ubertor/assets/199635/8c4fadf6-7a13-4ba3-925e-281b1877f5b1)

Contact:

![2023-08-23_21-42_1](https://github.com/tremby/markwiens.ca-ubertor/assets/199635/9b0bdad5-07b7-4f48-a287-72af175a0ccd)

Compare with selling:

![2023-08-23_21-38](https://github.com/tremby/markwiens.ca-ubertor/assets/199635/484f15d0-1db5-4260-8cd5-f8d7b480c0c1)
